build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.2.2"
```

Comment:
This out of the scope the commit, normally if you want to add somethings and not others in code you can use `git commit -up .` and enter `y` to add the change and `n` to skip the chnage.

Reply:
Sure